src/00_prepare_files_and_folders.R

```diff
@@ -0,0 +1,429 @@
+#--------------------------------------------
+#-----------  Load packages  ----------------
+#--------------------------------------------
+packages <- c("curl", "zen4R", "here", "sf", "terra")
```

Comment:
Maybe add `rnaturalearth` to the list of packages. At least for me it interupted the flow.